src/org/opendatakit/common/cli/CustomHelpFormatter.java

```diff
@@ -53,7 +78,7 @@ private static void printRequiredParams(Map<String, String> helpLinesPerShortcod
   }
 
   private static void printOptionalParams(Map<String, String> helpLinesPerShortcode, Operation operation) {
-    System.out.println("  (optionally)");
+    System.out.println("Optional params for -" + operation.param.shortCode + " operation:");
```

Comment:
I'd add a `System.out.println("\n")` before this.

Reply:
I'd vote for not adding the blank line.

I tried it and went for keeping everything together because it felt more confusing as blank lines separate the different operations and their params.